src/lib/utils/tokens.ts

```diff
@@ -142,11 +143,16 @@ export function isAvailableForFlashMint(token: Token): boolean {
 }
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-export function isAvailableForIssuance(inputToken: Token, _: Token): boolean {
+export function isAvailableForIssuance(
+  inputToken: Token,
+  outputToken: Token,
+): boolean {
   return (
     inputToken.symbol === BedIndex.symbol ||
     inputToken.symbol === GitcoinStakedETHIndex.symbol ||
-    inputToken.symbol === LeveragedRethStakingYield.symbol
+    inputToken.symbol === LeveragedRethStakingYield.symbol ||
+    inputToken.symbol === RealWorldAssetIndex.symbol ||
+    outputToken.symbol === RealWorldAssetIndex.symbol
```

Comment:
Curious - why is it `true` here if it's not for hyETH? Or was hyETH here before but got removed?

Reply:
Because for presale the tokens use the issuance and once live they use flash minting.